contracts/migration/localhost/Aggregator/20230325094038_ETH-USDT.json

```diff
@@ -1,15 +1,20 @@
 {
   "deploy": {
     "name": "ETH-USDT",
+    "paymentAmount": 0,
```

Comment:
Can you please remove the `paymentAmount`, `depositAmount` and `addedAdmins`? They were removed from the latest on-chain code.